plugins.json

```diff
@@ -2499,6 +2499,62 @@
     },
     "updated_at": "2019-06-12T19:04:44Z"
   },
+  {
+    "homepage_url": "",
```

Comment:
Can you file a parallel ticket to update our metadata in the repository's `plugin.json`, as well as a ticket to get an icon designed?

Reply:
https://mattermost.atlassian.net/browse/MM-30960
https://mattermost.atlassian.net/browse/MM-30961